### PR TITLE
Warn about security risks of the recommended libvirt install configuration

### PR DESCRIPTION
**Pull request by berrange**
The libvirt installer documentation is recommending a highly insecure configuration of libvirtd, with no auth or encryption, listening on all IP addresses, which gives clients prives equiv to a root shell. It does give some sense of this being an insecure config, but doesn't spell out the full implications to users following the doc. So this adds a much more explicit warning about the risks to fully inform users.

A few other minor improvements were made too.

### CODE REVIEW

**Comment by berrange** [comment]
@cfergeau @zeenix who were looking at options for enabling a more secure encryption & auth scheme for the libvirt installer.

File: docs/dev/libvirt/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ repository to ensure you get a new enough version of qemu-kvm.
 On Fedora, CentOS/RHEL:
 
 ```sh
-sudo yum install libvirt libvirt-devel libvirt-daemon-kvm qemu-kvm
+sudo yum install libvirt-devel libvirt-daemon-kvm
```
Comment (cfergeau) [inline docs/dev/libvirt/README.md:33]: I would add `libvirt-client` to that list, or add a mention that it's needed if you need `virsh`. Or is it already a dependency of `libvirt-devel`/`libvirt-daemon-kvm`?

Comment (berrange) [reply docs/dev/libvirt/README.md]: Yes, good point, I've done that

**Comment by cfergeau** [comment]
@zeenix can you take a look/merge this?

**Pull request merged**